### ISSUE TITLE
Reuse auth token for accessing OCIR with option of using sealed secrets

### ISSUE THREAD

**Issue by livespotty** [oracle-terraform-modules/terraform-oci-oke#153]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

The feature request is to have auth token reused while generating cluster. Rather than creating new token everytime, this feature will enable reuse.
Optional: Sometimes the cluster is created via gitops and hence needed the secrets (auth token) to be stored as part of git. So, optionally if this can integrate with other tools such as sealed secrets https://github.com/bitnami-labs/sealed-secrets will be good.

### New or Affected Resource(s)

OCIR

### Potential Terraform Configuration

```hcl
# Copy-paste any Terraform configurations for how the requested feature may be used. 

https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/master/docs/configuration.adoc#configure-ocir-parameters

```

### References

https://github.com/bitnami-labs/sealed-secrets

**Comment by hyder** [comment]
@karthicgit

**Comment by hyder** [comment]
Auth Token reuse fixed in #161. Thanks @karthicgit for implementing and @livespotty for testing.
